src/main/java/com/invoiced/entity/BankAccount.java

```diff
@@ -0,0 +1,41 @@
+package com.invoiced.entity;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.invoiced.exception.EntityException;
+
+public class BankAccount extends PaymentSource {
+
+  @JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
It seems like you need to run the code formatter again since these properties don't have any whitespace between?

Reply:
The formatter really can't decide on whether to put whitespace there or not. I was going to ask which you prefer; I prefer whitespace being there. But last time I ran the formatter it removed it all

Reply:
Ok that wasn't clear since the rest of the PR is adding whitespace. I'd go with the formatter and not fight it. If there is a different formatter you would recommend that does give whitespace we can adopt it in lieu of google code formatter.

Reply:
Alright, cool, no need to fight the formatter. In that case it's good to merge as-is.